examples/android/build.gradle

```diff
@@ -1,13 +1,13 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
-    
+
     repositories {
         google()
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.0'
+        classpath 'com.android.tools.build:gradle:3.1.0'
```

Comment:
What's the difference between 3.0 and 3.1?

Reply:
It's basically the build system plugin that's downloaded and used. It's not necessarily mandatory to change, but might improve compatibility.